User_Guides/getting_started/Getting_Started_SS3.Rmd

```diff
@@ -72,7 +72,7 @@ Many output text files are created during a model run. The most useful output fi
 
```

Comment:
@Rick-Methot-NOAA Should we remove info about SSI from the SS3 documentation materials?